satellite/console/consoleweb/consoleql/mail.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package consoleql
+
+const (
+	// ActivationPath is key for pass which handles account activation
+	ActivationPath = "activationPath"
+
+	// ActivationSubject activation email subject
+	ActivationSubject = "Activate your email"
+	// InvitationSubject invitation email subject
+	InvitationSubject = ""
+	// ForgotPasswordSubject forgot password email subject
+	ForgotPasswordSubject = ""
+)
+
+// MailTemplate is implementation of satellite/mailservice.Template interface
+type MailTemplate struct {
```

Comment:
If you make these methods which have hard-coded values then the code will look nicer.

```
func (AccountActivationEmail) Template() string { return "activation-email" }
```